spec/models/whitehall_asset_spec.rb

```diff
@@ -75,9 +75,7 @@
 
     context 'when updating asset' do
       it 'cannot be set' do
-        asset.legacy_url_path = '/government/uploads/another-asset.png'
-        asset.save!
-        expect(asset.reload.legacy_url_path).to eq('/government/uploads/asset.png')
+        expect { asset.legacy_url_path = '/government/uploads/another-asset.png' }.to raise_error(Mongoid::Errors::ReadonlyAttribute)
```

Comment:
This also looks like a change in behaviour. Is it related to upgrading Mongo?

Reply:
Yeah - I've added an explanation to the commit message as to why these tests have changed. Mongoid now throws exceptions when trying to update a readonly attribute so the tests needed to check that the exception was thrown